avatica-shaded/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <artifactId>kafka-connect-storage-common-parent</artifactId>
+    <groupId>io.confluent</groupId>
+    <version>10.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>kafka-connect-storage-common-avatica-shaded</artifactId>
+  <name>kafka-connect-storage-common-avatica-shaded</name>
+  <description>avatica shaded package to replace jackson dependencies without CVEs</description>

```

Comment:
Can you explain how this file is used? It has the same artifactId as the `avatica-shaded/pom.xml` file, and it's not explicitly referenced as far as I can see.

Reply:
Removed. Added by mistake.